compiler/noirc_frontend/src/ast/statement.rs

```diff
@@ -40,7 +40,7 @@ pub enum StatementKind {
     Break,
     Continue,
     /// This statement should be executed at compile-time
-    Comptime(Box<StatementKind>),
```

Comment:
I kind of prefer just Comptime without the capitalized T was this done for an explicit reason?

Reply:
I was worried spellcheck would flag "comptime" as not a word. I can revert it if you want. I think we have an exception for it already anyway

Reply:
I find a bit cleaner no? And we can just add it to spell check then

Reply:
This is non-blocking either way